README.md

```diff
@@ -1,14 +1,18 @@
 # MoveIt! Tutorials
 
-This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/indigo/api/moveit_tutorials/html/
+This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
```

Comment:
...so yes, the buildfarm will generate documentation for kinetic

its in queue: http://build.ros.org/job/Kdoc__moveit_tutorials__ubuntu_xenial_amd64/